Bigtable/src/DataClient.php

```diff
@@ -322,4 +324,83 @@ public function readRow($rowKey, array $options = [])
             ->readAll()
             ->current();
     }
+
+    /**
+     * Modifies a row atomically on the server. The method reads the latest
+     * existing timestamp and value from the specified columns and writes a new
+     * entry based on pre-defined read/modify/write rules. The new value for the
+     * timestamp is the greater of the existing timestamp or the current server
+     * time. The method returns the new contents of all modified cells.
+     *
+     * Example:
+     * ```
+     * use Google\Cloud\Bigtable\ReadModifyWriteRowRules;
+     *
+     * $rules = (new ReadModifyWriteRowRules)
+     *     ->append('cf1', 'cq1', 'value12');
+     * $row = $dataClient->readModifyWriteRow('rk1', $rules);
+     *
+     * print_r($row);
+     * ```
+     *
+     * //Increments value
```

Comment:
Just a little nit: For examples beyond the first one, this comment should be inside the fence in order to be picked up correctly by the documentation generator:

```diff
-    * //Increments value
-    * ```
+    * ```
+    * // Increments value
```